examples/example-scripttask-ruby/nodes/rb/runScript.rb

```diff
@@ -0,0 +1,2 @@
+print "Hello World from Ruby workflow!!"
```

Comment:
```suggestion
print "\nHello World from Ruby workflow!!\n"
```

I think We should add new lines to see this more easily in the logs.